vignettes/intro.Rmd

```diff
@@ -27,6 +28,238 @@ uirender(uiinput(icon("dog"),
 ), width = "200px", height = "100px")
 ```
 
+## Create a Shiny App
+Let's create a Shiny app with `semanticPage()` and just a `h1` title.
 
-To define a new page use:
+```{r echo=TRUE, include=TRUE}
+ui <- semanticPage(
+  title = "My first page",
+  h1("My page")
+)
+
+server <- function(input, output, session) {}
+
+shinyApp(ui, server)
+```
+
+Now let's add sidebar layout `sidebar_layout()`.
+
+```{r echo=TRUE, include=TRUE}
+ui <- semanticPage(
+  title = "My first page",
+  h1("My page"),
+  sidebar_layout(
+    sidebar_panel(),
+    main_panel()
+    )
+)
+
+server <- function(input, output, session) {}
+
+shinyApp(ui, server)
+```
+
+Next let's fill in the main panel `main_panel()` with a Semantic UI segment containing 2 cards.
+
+```{r}
+ui <- semanticPage(
+  title = "My first page",
+  h1("My page"),
+  sidebar_layout(
+    sidebar_panel(),
+    main_panel(
+      segment(
+         cards(
+           class = "two",
+           card(class = "red",
+             div(class = "content",
+                 div(class = "header", "Main title card 1"),
+                 div(class = "meta", "Sub title card 1"),
+                 div(class = "description", "More detail description card 1")
+             )
+           ),
+           card(class = "blue",
+             div(class = "content",
+                 div(class = "header", "Main title card 2"),
+                 div(class = "meta", "Sub title card 2"),
+                 div(class = "description", "More detail description card 2")
+                 )
+             )
+           )
+         )
+      )
+    )
+  )
+
+server <- function(input, output, session) {}
+
+shinyApp(ui, server)
+```
+
+Time to fill in sidebar `sidebar_panel()`, so let's add dropdown. We create a dropdown input using `dropdown_input()`.
```

Comment:
"a dropdown" and the "the dropdown", or simply "We create it"

Reply:
done